src/OpenApi/gen/XmlCommentGenerator.Emitter.cs

```diff
@@ -407,7 +407,7 @@ private static OpenApiParameter UnwrapOpenApiParameter(IOpenApiParameter sourceP
             }
             else
             {
-                throw new InvalidOperationException("The input schema must be an OpenApiParameter or OpenApiParameterReference.");
+                throw new InvalidOperationException("The input schema must be an {nameof(OpenApiParameter)} or {nameof(OpenApiParameterReference)}.");
```

Comment:
```suggestion
                throw new InvalidOperationException($"The input schema must be an {nameof(OpenApiParameter)} or {nameof(OpenApiParameterReference)}.");
```

Reply:
omg I hate myself

Reply:
This one did make me feel bad 😅 when I spotted it in the snapshots

Reply:
it's ok I groaned harder on the other whitespace issue

I __really__ wish there was a nicer templating story for source generated files so you could catch these kinds of things more easily